api/tests/routes/backoffice/gdpr_user_extract_data_test.py

```diff
@@ -186,3 +188,45 @@ def test_no_file_in_bucket(self, authenticated_client):
         assert "L'extraction demandée existe mais aucune archive ne lui est associée" in html_parser.extract_alerts(
             redirection.data
         )
+
+
+class DeleteGdprUserExtractTest(PostEndpointHelper, StorageFolderManager):
+    endpoint = "backoffice_web.gdpr_extract.delete_gdpr_user_data_extract"
+    endpoint_kwargs = {"gdpr_id": 1}
+    needed_permission = perm_models.Permissions.EXTRACT_PUBLIC_ACCOUNT
+    storage_folder = settings.LOCAL_STORAGE_DIR / settings.GCP_GDPR_EXTRACT_BUCKET / settings.GCP_GDPR_EXTRACT_FOLDER
+
+    def test_delete_gdpr_user_extract(self, authenticated_client):
+        extract = users_factories.GdprUserDataExtractBeneficiaryFactory(dateProcessed=datetime.datetime.utcnow())
+        with open(self.storage_folder / f"{extract.id}.zip", "wb") as fp:
+            fp.write(b"[personal data compressed with deflate]")
```

Comment:
on a pas besoin du contenu ou que le fichier ai un contenu donc on peyut simplifier en créant juste un fichier vide
```suggestion
        open(self.storage_folder / f"{extract.id}.zip", 'wb').close()

```